### ISSUE TITLE
Introducing `rebase` to our daily workflow

### ISSUE THREAD

**Issue by lyzadanger** [cloudfour/guides#5]
As we get bigger and badder, I'm seeing a lot of chattery commits and a lot of redundant merge commits and merges overall. I'm interested in introducing a `rebase` policy, but wanted to get feedback. How do we explain the concept and how do we "sell" it, if it's something we'd like to encourage.

I think [this recent post](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/) is written mostly in plain English, but might be long for those who aren't particularly interested. Thoughts?

**Comment by derekshirk** [comment]
I've always liked the Atlassian `git` guides & tutorials. I find them well-written and easy to follow. 

- https://www.atlassian.com/git/tutorials/merging-vs-rebasing
- https://www.atlassian.com/git/tutorials/rewriting-history#git-rebase

**Comment by erikjung** [comment]
FWIW, since this was originally brought up, I've typically only used `rebase` in this situation:

- I have new a feature (`feature_b`) that is dependent on another new feature (`feature_a`).
- I will work on Feature A in a branch (`feature-new_feature_a`) and issue a PR for it.
- If I anticipate the Feature A PR taking a while to get merged, I'll branch off of it:
  ```bash
  git checkout feature-new_feature_a
  git checkout -b feature-new_feature_b
  ```
- Then if the Feature A PR is merged while I'm working on Feature B (**before pushing the branch**), I'll use `rebase` to base Feature B off of the branch that Feature A was merged into:
  ```bash
  git fetch dev
  git rebase dev
  git push origin feature-new_feature_b
  ```

**Comment by spaceninja** [comment]
This has been addressed: https://github.com/cloudfour/guides/tree/main/git#merging-in-general
